server/app.ts

```diff
@@ -10,7 +10,7 @@ app.use(express.static(`${__dirname}/public`));
 
 app.use(express.json({ limit: '10kb' }));
 
-app.use('/api/v1/users', userRouter);
+app.use('/', indexRouter);
```

Comment:
Once again. I don't really like the idea of indexRouter. That's making app.ts file just less and less understandable. I suggest that It should keep proper routers usage and proper endpoints for these routers.

Reply:
I can definitely agree that the code related to routing has definitely too much indirection. IMO a separate file for every endpoint and every router is an overkill...

What I'd propose is to have one main router inside of one file and then separate routers for the main entities inside the API. So for now this would be just tournaments. IMO the definition of particular endpoints for tournaments should not be in separate files. Keep in mind that the implementation for the request handlers should be minimal. It should just handle the HTTP stuff and call some business code that lies in a different component.

Reply:
We finally did it exactly this way

Reply:
It's little outdated. I / We move all routers logic to /routes/index.ts